src/App.test.tsx

```diff
@@ -30,4 +32,98 @@ describe('App component:', () => {
       expect(screen.getByText(/emotions category/i)).toBeInTheDocument();
     });
   });
+  // ***************************************************INTEGRATION TESTS**********************************************
+  describe('opens', () => {
+    it('sidebar on header button click', () => {
+      const renderComp = render(
+        <MemoryRouter initialEntries={['/']}>
+          <App />
+        </MemoryRouter>
+      );
+      const openButton = renderComp.container.getElementsByClassName('SideBarOpenButton');
+      // before user click
+      // now SideBar is not active
+      expect(renderComp.container.getElementsByClassName('SideBar active')[0]).toBeFalsy(); // is not active
```

Comment:
Please don't relay on the class names. Use `[data-testid='my-id']` https://testing-library.com/docs/queries/bytestid/